t/toml-test/valid/array-empty.t

```diff
@@ -2,8 +2,6 @@
 use utf8;
 use Test2::V0;
 use Data::Dumper;
-use DateTime;
-use DateTime::Format::RFC3339;
```

Comment:
Strangely, when I run the test builder on my machine with a current clone of the toml-test repo, this test, and many others like it, do not get re-generated (maybe I'm on a different version of toml-test?).

I kept the modified files since, if the test is staying, it should not depend DateTime. But I'm not sure why the builder is not picking this up.

Reply:
I just checked out your fork as well as the latest master from `BurntSushi/toml-test` in the same directory. I was able to update the files by running the test builder:

```
$ cd $HOME/tmp
$ git clone git@github.com:BurntSushi/toml-test.git
$ git clone git@github.com:jjatria/TOML-Tiny.git
$ cd TOML-Tiny
$ perl build-tests.pl ../toml-test .
$ git status

On branch master                                                                                         
Your branch is up to date with 'origin/master'.                                                                                                                                                                    
                                                                                                         
Changes not staged for commit:                                                                           
  (use "git add <file>..." to update what will be committed)                            
  (use "git restore <file>..." to discard changes in working directory)
        modified:   t/toml-test/valid/example.t                                                          
        modified:   t/toml-test/valid/implicit-and-explicit-after.t                   
        modified:   t/toml-test/valid/implicit-and-explicit-before.t                                                                                                                                               
        modified:   t/toml-test/valid/implicit-groups.t                  
        modified:   t/toml-test/valid/newline-crlf.t                       
                                                                                                         
Untracked files:                                                                                         
  (use "git add <file>..." to include in what will be committed)           
        t/toml-test/valid/empty-file.t                                                                                                                                                                             
        t/toml-test/valid/spec-example-1-compact.t                                                       
        t/toml-test/valid/spec-example-1.t
                                                                                                         
no changes added to commit (use "git add" and/or "git commit -a")
```

Reply:
Oh, yeah, that's what I did when I was testing things. What I meant was that, if I run the modified test builder on an otherwise fresh version of the repo, tests like `t/toml-test/valid/array-empty.t` keep the references to DateTime. Indeed: if I delete this test file, and run the test builder, this file does not get re-created.

Is this as expected? Or am I'm missing something?

Reply:
Perhaps that file is no longer in the `toml-test` repo? Try deleting everything out of `valid` and `invalid` and seeing if it recreates the file at all. It may just be that the source test has disappeared and is exposing some rough areas of a tool that was really just banged out as quickly as possible :P